dff/script/core/types.py

```diff
@@ -40,39 +40,56 @@
 # TODO: change example
 
 
-# TODO: add description for each stage of ActorStage
 class ActorStage(Enum):
     """
     The class which holds keys for the handlers. These keys are used
-    for the actions of :py:class:`~dff.script.Actor`.
+    for the actions of :py:class:`.Actor`. Each stage represents
+    a specific step in the conversation flow. Here is a brief description
+    of each stage.
     """
 
-    #: This stage is used for the context initializing.
+    #: This stage is used for the context initialization.
+    #: It involves setting up the conversation context.
     CONTEXT_INIT = auto()
 
-    #: This stage is used to get the previous node.
+    #: This stage is used to retrieve the previous node.
+    #: It involves identifying the previous node in the conversation history.
     GET_PREVIOUS_NODE = auto()
 
-    #: This stage is used for rewriting the previous node.
+    #: This stage is used to rewrite the previous node.
+    #: It involves updating the previous node in the conversation history
+    #: to reflect any changes made during the current conversation turn.
     REWRITE_PREVIOUS_NODE = auto()
 
-    #: This stage is used for running pre-transitions processing.
+    #: This stage is used for running pre-response processing.
+    #: It involves performing any necessary pre-processing tasks.
     RUN_PRE_TRANSITIONS_PROCESSING = auto()
 
-    #: This stage is used to get true labels.
+    #: This stage is used to retrieve the true labels.
+    #: It involves determining the correct label to take based
+    #: on the current conversation context.
     GET_TRUE_LABELS = auto()
 
-    #: This stage is used to get next node.
+    #: This stage is used to retrieve the next node.
+    #: It involves identifying the next node in the conversation flow.
```

Comment:
Again, this is repetitive. Maybe just `This stage is used to retrieve the next node in the conversation flow.`?